ethpm_cli/commands/registry.py

```diff
@@ -203,3 +216,15 @@ def generate_registry_store_data(
         "alias": alias,
         "active": activate,
     }
+
+
+def display_packages(all_package_names: Iterable[str], config: Config) -> None:
+    for package_name in all_package_names:
+        all_releases = config.w3.pm.get_all_package_releases(package_name)
+        cli_logger.info(f"Retrieving all releases for {bold_blue(package_name)}: \n")
+        for version, manifest_uri in all_releases:
+            cli_logger.info(f"{bold_green(version)} --- ({bold_white(manifest_uri)})")
+        cli_logger.info(
+            f"Total releases: {config.w3.pm.get_release_count(package_name)}\n"
```

Comment:
You should be able to get away without this `w3` call and just use `len(all_releases)` to get the count.